### PR TITLE
Config Module

### PR DESCRIPTION
**Pull request by yee2542**

### CODE REVIEW

**Review by yee2542 — COMMENTED** [review]
Gogo

File: src/config/config.enum.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export enum ConfigEnum {
+  DATABASE_CONNECTION = 'DATABASE_CONNECTION',
+  DATABASE_USERAME = 'DATABASE_USERAME',
```
Comment (octobrekids) [inline src/config/config.enum.ts:3]: Please rename to `DATABASE_USERNAME`

File: src/config/config.providers.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { ConfigurationInterface } from './config.interface';
+import { ProcessConfigType } from './process.config.type';
+
+export const configProviders: Provider[] = [
+  {
+    provide: APP_CONFIG,
+    useValue: ((): ConfigurationInterface => {
+      const {
+        DATABASE_CONNECTION,
+        DATABASE_USERAME,
```
Comment (octobrekids) [inline src/config/config.providers.ts:12]: Please rename to `DATABASE_USERNAME`

**Review by octobrekids — COMMENTED** [review]
Please check

**Review by octobrekids — APPROVED** [review]
Seen

**Pull request merged**